src/core/hooks/InvitationsGraphQL/useRemoveInvitation.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react';
+
+import { IInvitationDataProps } from 'context';
+import { useRemoveInvitationMutation } from 'core/graphql';
+import { allInvitationsQuery } from 'core/operations';
+
+export interface IRemoveInvitation {
+  invitations: IInvitationDataProps[],
+  companyId: string
+}
+
+export const useRemoveInvitation = (): [(props: IRemoveInvitation) => void, (invitations: IInvitationDataProps[]) => void] => {
```

Comment:
Move out return type.

Reply:
`(props: IRemoveInvitationProps): RemoveInvitationHookType`
For example.

Reply:
Pls, see commit below, is that what you mean?